index.js

```diff
@@ -35,8 +35,9 @@ class HandlebarsRenderer {
     * @param {Object} themeSettings - Theme settings (configuration), passed to helpers
     * @param {String} hbVersion - Which version of handlebars to use. One of ['v3', 'v4'] - defaults to 'v3'.
     * @param {Object} logger - A console-like object to use for logging
+    * @param {String} logLevel - log level which will be overriden by renderer
     */
-    constructor(siteSettings, themeSettings, hbVersion, logger = console) {
+    constructor(siteSettings, themeSettings, hbVersion, logger = console, logLevel = 'info') {
```

Comment:
Do we want to make this configurable env var? that would let us to adjust the level to debug in like integration if we wanted additional logging verbosity there.